### ISSUE TITLE
[Reactive] [Android] Spinner delay

### ISSUE THREAD

**Issue by DiegoBillares** [unoplatform/uno.extensions#742]
## Current behavior

The app takes a delay to show spinner.

## Expected behavior

The app should show the spinner like every element on the page, i.e. The spinner should appear at the same time with the text box title.

## How to reproduce it (as minimally and precisely as possible)

- Open app
- The current view show the bug

## Environment

Nuget Package: Uno.Extensions.Reactive.WinUI

Package Version(s): 2.3.0-dev.10

Affected platform(s):

- [ ] iOS
- [x] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [x] 2022 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

## Anything else we need to know?
[EmptySampleApp.zip](https://github.com/unoplatform/uno.extensions/files/9599014/EmptySampleApp.zip)

**Comment by dr1rrb** [comment]
Is this an Android only issue, or you tested only on Android?

If it's on all platforms, then it's most probably an expected behavior: As nowadays networks are pretty fast also on mobile, we try to avoid a flickering loading wheel by deferring it's appearance by 250ms in the default FeedView's template: https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive.UI/View/FeedView.xaml#L44

If it does not fit the needs of your project / API, you can change / disable it by changing the template.

**Comment by Soap-141** [comment]
@dr1rrb It's definitely more than 250ms on Android. It's literally more than a second sometime if I remember correctly.

**Comment by francoistanguay** [comment]
Is this the case in the repro sample? Can we have a video as well maybe?

**Comment by DiegoBillares** [comment]
Hello, I published a thread on microsoft teams about this: 
https://teams.microsoft.com/l/message/19:84af563de4e04bc6bf70488501229bdb@thread.tacv2/1664572892192?tenantId=6d53ef61-b6d1-4150-ae0b-43b90e75e0cd&groupId=4f5def50-a8cd-4c94-a30a-016a46cd5f6b&parentMessageId=1664572892192&teamName=Uno%20Collab&channelName=General&createdTime=1664572892192&allowXTenantAccess=false

Although, I can upload here one video that I attached on the thread:

Uploading Android Emulator - nexus_4_-_api_30_5554 2022-09-30 18-19-11.mp4…

**Comment by francoistanguay** [comment]
I dont see how the video is related to the issue. @Soap-141  we need clearer info.

**Comment by DiegoBillares** [comment]
Now yes, sorry I had a mistake

**Comment by francoistanguay** [comment]
Looks like <250ms to me. Can you update the style like @dr1rrb  suggested and see if it's aligned with your expectaitons?

**Comment by DiegoBillares** [comment]
I changed the delay to 0ms but it takes a tiny time to appear. is that fine?

https://user-images.githubusercontent.com/94696299/193632506-0c840ad0-6428-4b90-851f-8c9ab0a0ec39.mp4

**Comment by francoistanguay** [comment]
Is this running in Release? With AOT?

**Comment by DiegoBillares** [comment]
No, in debugging

**Comment by francoistanguay** [comment]
Yeah, then it doesnt mean anything. Will wait for you to check in Release, with AOT before checking for any perf issue.

**Comment by DiegoBillares** [comment]
I executed in Release with AOT and it seems the same I think:
 

https://user-images.githubusercontent.com/94696299/193644641-e642244e-2dbc-4796-94dc-93bc6763fbc0.mp4

**Comment by jeromelaban** [comment]
The emulator seems very slow. Which is that? Is it using x86_64?

**Comment by DiegoBillares** [comment]
Yes, isn't it fine?

**Comment by jeromelaban** [comment]
Do you have Hyper-V enabled on your machine as well? (Just [to be sure](https://learn.microsoft.com/en-us/dotnet/maui/android/emulator/hardware-acceleration))

**Comment by DiegoBillares** [comment]
Yes:
![image](https://user-images.githubusercontent.com/94696299/193655216-1a8d5daf-009c-4cf8-a381-f95e10d32619.png)

**Comment by jeromelaban** [comment]
Thanks. Can you also share the device logs when the app starts?

**Comment by DiegoBillares** [comment]
Let me know if it's this:

[BuildOutput.txt](https://github.com/unoplatform/uno.extensions/files/9700267/BuildOutput.txt)

**Comment by jeromelaban** [comment]
Thanks, those are the build output. For the devices logs, you'll need to use `adb logcat`.

**Comment by DiegoBillares** [comment]
All right, let me know if this is enough 
[LogCat.txt](https://github.com/unoplatform/uno.extensions/files/9707461/LogCat.txt)
